include/misc/wallinTerran.hpp

```diff
@@ -80,8 +80,8 @@ namespace ghost
   vector< Building > makeTerranBuildings()
   {
     // Academy
-    Building a1 = factoryTerranBuilding("Academy");
-    Building a2 = factoryTerranBuilding("Academy");
+//     Building a1 = factoryTerranBuilding("Academy");
```

Comment:
Right.
I am not sure making dynamic the building list brings something: why adding or removing buildings? If you need it, it means you want to solve another problem instance.